exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,12 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+if (danielsRole === "mentor") {
+  console.log("Hi, I'm " + name + ", I'm a " + danielsRole + ".");
+}
+else if (danielsRole === "student") {
+  console.log("Hi, I'm " + name + ", I'm a " + danielsRole + ".");
+}
```

Comment:
good job Jaspreet, i liked that you didnt forget to write name "danielsRole"

Reply:
thankyou so much nage to review my code